cmd/kops/update_cluster.go

```diff
@@ -172,6 +175,20 @@ func RunUpdateCluster(f *util.Factory, clusterName string, out io.Writer, c *Upd
 		glog.Infof("Using SSH public key: %v\n", c.SSHPublicKey)
 	}
 
+	var phase cloudup.Phase
```

Comment:
Should we have the phase options lower, say in create.go?  I will probably need `kops create -f mycluster.yaml --phase iam`.